package.json

```diff
@@ -5,10 +5,13 @@
   "module": "index.js",
   "main": "dist/index.js",
   "scripts": {
-    "build": "node ./compileComponents.js && webpack",
-    "prepublish": "npm run build && npm run test ",
+    "build": "npm run build:umd && npm run build:es",
+    "build:umd": "cross-env NODE_ENV=production webpack --config webpack.config.umd.js",
+    "build:es": "cross-env NODE_ENV=production babel . --out-dir . --extensions \".jsx\" --ignore node_modules,sample && npm run copycss",
+    "copycss": "node copyCss.js",
```

Comment:
For ES6, we weren't using Webpack anyway, just a babel command to compile the jsx and earlier-stage-ES6. This makes it clearer and puts it into one command. I'm somewhat copying what Brian is doing with react-virtualized [here](https://github.com/bvaughn/react-virtualized/blob/master/package.json).